core-ui/src/shared/components/StatsGraph/StatsPanel.js

```diff
@@ -136,6 +144,10 @@ export function StatsPanel({ type, ...props }) {
     return '';
   }
 
+  let path = features.PROMETHEUS?.config?.path;
```

Comment:
wouldn't it be better to read the path inside the `usePrometheus` hook? It's read from microfrontend context anyway - passing the path through these components seems to only add unnecessary complexity